cmd/kubepug.go

```diff
@@ -84,6 +103,28 @@ func runPug(cmd *cobra.Command, args []string) error {
 	log.Debugf("Starting Kubepug with configs: %+v", config)
 	kubepug := lib.NewKubepug(config)
 
+	if config.Monitor {
+		done := make(chan os.Signal)
+		signal.Notify(done, os.Interrupt)
+		go func() {
+			http.Handle("/metrics", promhttp.Handler())
+			http.ListenAndServe("0.0.0.0:8888", nil)
```

Comment:
Just for security, can we make the binding address and the port also configurable?